app/index.html

```diff
@@ -189,30 +189,36 @@ <h2>PWA Starter Kit</h2>
         </a>
-        <a class="onload-fadein product-grid-item left" href="https://github.com/material-components/material-components-web-components" style="animation-delay: .25s; background: #00D67C; background: linear-gradient(315deg, rgba(255,255,255,0.5) 0%, rgba(255,255,255,0) 100%), linear-gradient(315deg, #20FFA7 0%, #00D67C 100%)">
-          <div class="thumb" style="background-image:url(/images/thumbnails/thumb_material.svg); padding-top: 48%; margin: 10px 15%;"></div>
+        <a class="onload-fadein product-grid-item left" href="https://lit-html.polymer-project.org" style="animation-delay: .3s; background: #FF2F2F; background: linear-gradient(315deg, rgba(255,255,255,0.5) 0%, rgba(255,255,255,0) 100%), linear-gradient(315deg, #FA5A5A 0%, #FF2F2F 100%)">
+          <div class="thumb" style="background-image:url(/images/thumbnails/thumb_lit-html.svg); padding-top: 45%"></div>
+          <h2>lit-html</h2>
+          <p>An efficient, expressive, extensible HTML templating library for JavaScript.</p>
+          <span class="status">Prerelease</span>
+        </a>
+        <a class="onload-fadein product-grid-item right" href="https://github.com/material-components/material-components-web-components" style="animation-delay: .25s; background: #00D67C; background: linear-gradient(315deg, rgba(255,255,255,0.5) 0%, rgba(255,255,255,0) 100%), linear-gradient(315deg, #20FFA7 0%, #00D67C 100%)">
+          <div class="thumb" style="background-image:url(/images/thumbnails/thumb_material.svg); padding-top: 46%; margin: 10px 15%;"></div>
           <h2>Material Web Components</h2>
           <p>High-fidelity realizations of Google's Material Design spec that work anywhere on the web.</p>
           <span class="status">Prerelease</span>
         </a>
-        <a class="onload-fadein product-grid-item right" href="https://lit-html.polymer-project.org" style="animation-delay: .3s; background: #FF2F2F; background: linear-gradient(315deg, rgba(255,255,255,0.5) 0%, rgba(255,255,255,0) 100%), linear-gradient(315deg, #FA5A5A 0%, #FF2F2F 100%)">
-          <div class="thumb" style="background-image:url(/images/thumbnails/thumb_lit-html.svg); padding-top: 41%"></div>
-          <h2>lit-html</h2>
-          <p>An efficient, expressive, extensible HTML templating library for JavaScript.</p>
+        <a class="onload-fadein product-grid-item left" href="https://github.com/material-components/material-components-web-components" style="animation-delay: .25s; background: #00D67C; background: linear-gradient(315deg, rgba(255,255,255,0.5) 0%, rgba(255,255,255,0) 100%), linear-gradient(315deg, #FF4ED4 0%, #FF00C1 100%)">
+          <div class="thumb" style="background-image:url(/images/thumbnails/thumb_polyfills.svg); padding-top: 35%; margin: 12px 12%;"></div>
+          <h2>Web Components Polyfills</h2>
+          <p>A suite of polyfills supporting the Web Components specs.</p>
           <span class="status">Prerelease</span>
```

Comment:
For the Web Components Polyfills, let's change the status label to `Maintenance`  and href to [https://github.com/WebComponents/webcomponentsjs](https://github.com/WebComponents/webcomponentsjs)

cc'd @graynorton to confirm

Reply:
👍